code/obj/machinery/manufacturer.dm

```diff
@@ -26,62 +27,93 @@ TYPEINFO(/obj/machinery/manufacturer)
 	density = TRUE
 	anchored = ANCHORED
 	power_usage = 200
-	// req_access is used to lock out specific featurs and not limit deconstruciton therefore DECON_NO_ACCESS is required
+
+	/// req_access is used to lock out specific featurs and not limit deconstruciton therefore DECON_NO_ACCESS is required
 	req_access = list(access_heads)
 	event_handler_flags = NO_MOUSEDROP_QOL
 	deconstruct_flags = DECON_SCREWDRIVER | DECON_WRENCH | DECON_WELDER | DECON_WIRECUTTERS | DECON_MULTITOOL | DECON_NO_ACCESS
 	flags = NOSPLASH | FLUID_SUBMERGE
 	layer = STORAGE_LAYER
-	// General stuff
+	/* General stuff */
 	var/health = 100
-	var/supplemental_desc = null //! appended in get_desc() to the base description, to make subtype definitions cleaner
-	var/mode = MODE_READY //! the current status of the machine. Ready, working, or halt are all modes used currently.
+	/// Appended in get_desc() to the base description, to make subtype definitions cleaner
+	var/supplemental_desc = null
+	/// The current status of the machine.
+	/// "ready" / MODE_READY - The machine is ready to produce more blueprints.
+	/// "working" / MODE_WORKING - The machine is currently producing a blueprint.
+	/// "halt" / MODE_HALT - The machine stopped due to some problem arising.
+	var/mode = MODE_READY
+	/// A somewhat legacy variable to output silent yet visible errors to the user in the UI.
+	/// Current uses include when there is a lack of materials, an invalid blueprint, and when there is not enough manudrive uses.
 	var/error = null
-	var/active_power_consumption = 0 //! How much power is consumed while active? This is determined automatically when the unit starts a production cycle
+	/// How much power is consumed while active. This is determined automatically when the unit starts a production cycle
+	var/active_power_consumption = 0
 	var/panel_open = FALSE
+	/// The stage of dismantlement this machine is currently at.
```

Comment:
I would argue this comment adds nothing without explaining the number progression.

Reply:
resolved, explained progression and added defines to clarify elsewhere in usage